components/properties/src/props.rs

```diff
@@ -177,6 +179,7 @@ pub enum GeneralCategory {
 ///
 /// See `UCharCategory` and `U_GET_GC_MASK` in ICU4C.
 #[derive(Copy, Clone, PartialEq, Debug, Eq)]
+#[allow(clippy::exhaustive_structs)] // this type is stable
```

Comment:
Suggestion, here and elsewhere: I might tweak the comment for newtypes. This is a class of structs that are exhaustive.

```suggestion
#[allow(clippy::exhaustive_structs)] // newtype
```